xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -469,11 +475,71 @@ int CPVRChannelGroup::LoadFromDb()
   if (!database)
     return -1;
 
-  int iChannelCount = Size();
+  const int iChannelCount = Size();
+  const std::vector<std::shared_ptr<CPVRChannelGroupMember>> results = database->Get(*this);
+
+  std::vector<std::shared_ptr<CPVRChannelGroupMember>> membersToDelete;
+  if (!results.empty())
+  {
+    CSingleLock lock(m_critSection);
+    for (const auto& member : results)
+    {
+      // Consistency check.
+      if (member->ClientID() > 0 && member->ChannelUID() > 0 && member->IsRadio() == IsRadio())
+      {
+        m_sortedMembers.emplace_back(member);
+        m_members.emplace(
+            std::make_pair(std::make_pair(member->ClientID(), member->ChannelUID()), member));
```

Comment:
superfluous `std::make_pair` here, we only need one :) emplace will make the std::pair by forwarding its arguments

`m_members.emplace(std::make_pair(member->ClientID(), member->ChannelUID()), member);`

Reply:
Nice. Did not know this. Will change.